arm-servicefabric/2016-01-28/swagger/servicefabric.json

```diff
@@ -2974,7 +2974,7 @@
     "EventHealthEvaluation": {
       "type": "object",
       "allOf": [{"$ref": "#/definitions/HealthEvaluation"}],
-      "x-ms-discriminator-value": "1",
+      "x-ms-discriminator-value": "EventHealthEvaluation",
```

Comment:
Wait. As per the documentation for ["x-ms-discriminator-value"](https://github.com/Azure/autorest/blob/master/Documentation/swagger-extensions.md#x-ms-discriminator-value), it is used for specifying the value on wire. If your value on the wire is same as the model name then why do you need this extension ?